pkg/cmd/roachtest/tests/acceptance.go

```diff
@@ -97,15 +85,25 @@ func registerAcceptance(r registry.Registry) {
 				numNodes = tc.numNodes
 			}
 
-			spec := specTemplate
-			spec.Owner = owner
-			spec.Cluster = r.MakeClusterSpec(numNodes)
-			spec.Skip = tc.skip
-			spec.Name = specTemplate.Name + "/" + tc.name
+			spec := registry.TestSpec{
+				// NB: teamcity-post-failures.py relies on the acceptance tests
+				// being named acceptance/<testname> and will avoid posting a
+				// blank issue for the "acceptance" parent test. Make sure to
+				// teach that script (if it's still used at that point) should
+				// this naming scheme ever change (or issues such as #33519)
+				// will be posted.
```

Comment:
Since we're here, do you mind deleting this entire comment? It no longer applies.